modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -15,6 +15,15 @@
 namespace opencv_test
 {
 
+namespace
+{
+template <typename Elem>
+inline bool compareVectorsAbsExact(std::vector<Elem> outOCV, std::vector<Elem> outGAPI)
+{
+    return AbsExactVector<Elem>().to_compare_f()(outOCV, outGAPI);
```

Comment:
Functon with the one call inside looks extra

Reply:
The idea is to cover the "templateness" of `AbsExactVector`.
As you can see, I don't have to specify the template argument in tests checks because of this function